docker-compose.yml

```diff
@@ -1,11 +1,10 @@
 version: '3.1'
 
 services:
-  data-base:
+  db-postgres:
     image: postgres:latest
-    container_name: db-postgres
     restart: always
     environment:
       POSTGRES_PASSWORD: password
     ports:
-      - 5433:5432
+      - 5433:5432
```

Comment:
Please do not forget to add newline at the end of the file. It can affect git diff in future changes.